main.go

```diff
@@ -3,15 +3,35 @@ package main
 import (
 	"bufio"
 	"bytes"
+	"flag"
 	"fmt"
 	"io"
 	"os"
 )
 
 func main() {
+	uniqueParameter := flag.Bool("u", false, "allows you to print only unique lines.")
+	flag.Parse()
+
 	buf := bufio.NewReader(os.Stdin)
-	var checkEquality []byte
+	checkEquality := []byte("")
```

Comment:
What's this checkEquality change for?

Reply:
i had to change it so i can use it later as an if condition

Reply:
Really? Does the definition matter in you case? Please verify.

Reply:
yes it does becuase i cant compare checkEquality properly if its not initialized in the first iteration of my for loop